openassessment/templates/openassessmentblock/grade/oa_assessment_feedback.html

```diff
@@ -4,21 +4,21 @@
         <li class="answer feedback">
             {% if assessment.individual_assessments %}
                 {% for individual_assessment in assessment.individual_assessments %}
-                    {% if individual_assessment.feedback %}
-                        <div class="answer_title">
-                            <span class="answer__source">
-                                {%  if individual_assessment.option %}
-                                    {% blocktrans with title=individual_assessment.title grade=individual_assessment.option.label start_tag='<span class="answer__source__value">'|safe end_tag="</span>"|safe %}
-                                        {{ start_tag }}{{ title }}{{ end_tag }} - {{ grade }}
-                                    {% endblocktrans %}
-                                {% else %}
-                                    <span class="answer__source__value">
-                                        {{ individual_assessment.title }}
-                                    </span>
-                                {% endif %}
-                            </span>
-                        </div>
+                    <div class="answer_title">
+                        <span class="answer__source">
+                            {%  if individual_assessment.option %}
+                                {% blocktrans with title=individual_assessment.title grade=individual_assessment.option.label start_tag='<span class="answer__source__value">'|safe end_tag="</span>"|safe %}
+                                    {{ start_tag }}{{ title }}{{ end_tag }} - {{ grade }}
+                                {% endblocktrans %}
+                            {% else %}
```

Comment:
Perhaps this else should also check that individual_assessment.feedback exists to avoid the "empty header" issue?